<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLienzo</class>
 <widget class="QDialog" name="DialogLienzo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layoutPintar">
    <item>
     <widget class="QLabel" name="lblComando">
      <property name="text">
       <string>Comando</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtComando">
      <property name="placeholderText">
       <string>&lt;COLOR&gt;,([&lt;i&gt;,&lt;j&gt;]|[&lt;de i&gt;,&lt;de j&gt;,&lt;a i&gt;,&lt;a j&gt;]</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnComando">
      <property name="text">
       <string>Pintar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblImagen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>601</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>641</width>
     <height>104</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="grpX">
      <property name="title">
       <string>X</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblDe_x">
         <property name="text">
          <string>De:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spnBxDe_x"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblA_x">
         <property name="text">
          <string>A:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spnBxA_x"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grpY">
      <property name="title">
       <string>Y</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lblDe_y">
         <property name="text">
          <string>De:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spnBxDe_y"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblA_y">
         <property name="text">
          <string>A:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spnBxA_y"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCrear">
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGuardar">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEliminar">
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnGenerarPNG">
      <property name="text">
       <string>Generar PNG</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
